docs/ref/utils.txt

```diff
@@ -888,6 +888,30 @@ appropriate entities.
 
     If ``value`` is ``"你好 World"``, the output will be ``"你好-world"``.
 
+.. function:: format_lazy(format, *args, **kwargs)
+
+    .. versionadded:: 1.11
+
+    Lazy version of string format, useful when args and/or kwargs contains
+    lazy objects. First argument should be the string to be formatted.
+    For example::
```

Comment:
Maybe you want to add a few words about what this example does. As an inexperienced translation person, it's not extremely obvious.